docs/getting-started.md

```diff
@@ -81,7 +81,7 @@ option is the only required option in your configuration file.
 Now add a second page to your documentation:
 
 ```bash
-curl 'https://jaspervdj.be/lorem-markdownum/markdown.txt' > docs/about.md
+curl 'https://jaspervdj.be/lorem-markdownum/markdown.txt?no-external-links=on' > docs/about.md
```

Comment:
I have created https://tinyurl.com/lorem-markdownum-txt and https://dub.sh/lorem-markdownum shortlinks including the `no-external-links` param to avoid the long API URL with query param, but that'd need a -L flag to follow redirects and also kinda obscures what's the target URL so I ended up with just this original endpoint.

Nevertheless I've opened https://github.com/jaspervdj/lorem-markdownum/issues/14 to not generate random links, so if that one ever gets resolved I'll get back to this and change it back to default once safe;)